docs/reposado_operation.txt

```diff
@@ -68,6 +68,8 @@ Removing zzzz041-0532 (iTunes-10.2.2) from branch testing...
 
 would remove both iTunes 10.2.2.
 
+repoutil --purge-product all-deprecated removes all deprecated products.
+
```

Comment:
If it's going to be mentioned here at all, it needs more detail. It is documented here:
https://github.com/wdas/reposado/blob/master/docs/reference.txt#L159-L172